torchdata/datapipes/iter/load/aisio.py

```diff
@@ -51,13 +51,13 @@ class AISFileListerIterDataPipe(IterDataPipe[str]):
 
     Example:
         >>> from torchdata.datapipes.iter import IterableWrapper, AISFileLister
-        >>> ais_prefixes = IterableWrapper(['ais://bucket-name/folder/', 'aws:bucket-name/folder/', ...])
+        >>> ais_prefixes = IterableWrapper(['gcp://bucket-name/folder/', 'aws:bucket-name/folder/', 'ais://bucket-name/folder/', ...])
         >>> dp_ais_urls = AISFileLister(url='localhost:8080', source_datapipe=prefix)
-        >>> for d in dp_ais_urls:
+        >>> for url in dp_ais_urls:
         ...     pass
         >>> # Functional API
-        >>> dp_ais_urls = dp_ais_urls.list_files_by_ais(url='localhost:8080')
-        >>> for d in dp_ais_urls:
+        >>> dp_ais_urls = ais_prefixes.list_files_by_ais(url='localhost:8080')
```

Comment:
```suggestion
        >>> ais_prefixes = ais_prefixes.list_files_by_ais(url='localhost:8080')
```

Reply:
these would be complete URLs, right? why are we calling them prefixes? @NivekT

Reply:
Nevermind, I must have gotten something confused while reading.